<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>821</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="Logs_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>571</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Modified</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="Recover_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>500</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Recover</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Backup_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Backup</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="DatabaseLabel_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Database</string>
      </property>
     </widget>
     <widget class="QLabel" name="LogsLabel_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>See Logs below </string>
      </property>
     </widget>
     <widget class="QComboBox" name="Database_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>SQL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mongo</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="RefreshButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>330</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Logs</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="SelectFolder">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>330</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AWS_S3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>330</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload To AWS S3</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Encrypt">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>420</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Decrypt">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>420</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QComboBox" name="EncryptionDropDown">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>450</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Symmetric</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ceasar</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="Output_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>731</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="hostLabel_2">
         <property name="text">
          <string>host</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="hostLineEdit_2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="userLabel_2">
         <property name="text">
          <string>user</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="userLineEdit_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="passwordLabel_2">
         <property name="text">
          <string>password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="passwordLineEdit_2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="databaseLabel_2">
         <property name="text">
          <string>database</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="databaseLineEdit_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="QueryBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>541</width>
        <height>87</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Query_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
     <widget class="QLabel" name="QutputLabel_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
     <widget class="QLabel" name="QueryTabLabel_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Query Tab</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_12">
     <attribute name="title">
      <string>Tab 3</string>
     </attribute>
     <widget class="QPushButton" name="Convert">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QComboBox" name="DatabaseC_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Mongo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SQL</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="ConvertLabel_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>581</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Convert a Table From One Database To Other</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Save">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>370</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>691</width>
        <height>71</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Preview">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>370</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="PreviewBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>711</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="schemaData">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>711</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="schemaLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter you schema below ( use spaces in between )</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
